src/Carousel.tsx

```diff
@@ -23,14 +24,14 @@ export class Carousel extends React.Component<CarouselProps, CarouselState> {
     private scrollSyncTimer: number;
     private scrollDurationTimer: number;
     private animateDiv: HTMLDivElement;
-    private resizeListener = () => this.resize();
-    private scrollEventListener =() => this.onScroll();
+    private scrollEventListener = () => this.onScroll();
     private scrollAllowInterrupt = true;
 
     constructor(props: CarouselProps) {
         super(props);
 
         this.state = {
+            contentWidth: undefined,
```

Comment:
You need to `npm install` to get the latest version of TypeScript and the latest React type declarations.